reports/GoodsAndServiceTax/GSTR1View.js

```diff
@@ -1,10 +1,22 @@
 const title = 'GSTR 1';
 import baseConfig from './BaseViewConfig';
+import { DateTime } from 'luxon';
+
+const filterFields = baseConfig.filterFields;
+filterFields.forEach((field) => {
+  if (field.fieldname === 'toDate') {
+    field.default = DateTime.local().toISODate();
+  } 
+  else if(field.fieldname === 'fromDate') {
+    field.default = DateTime.local().minus({months: 3}).toISODate();
```

Comment:
I think these filters are not working correctly?

In the case of `toDate` and `fromDate` the same `default` field is set. can you please verify once?

Reply:
![image](https://user-images.githubusercontent.com/69912046/145706731-b77a26be-0096-4272-9936-5ce5fcd5bbdd.png)
Here in my machine this is what I get @ankitsinghaniyaz

Reply:
May be in the GST R2 page it might not be correct, so updating there as well

Reply:
Found a better way , I have updated the filters 💯